src/endpoints/users-private.js

```diff
@@ -58,26 +58,26 @@ router.get('/me', async (request, response) => {
 router.post('/change-avatar', jsonParser, async (request, response) => {
     try {
         if (!request.body.handle) {
-            console.log('Change avatar failed: Missing required fields');
+            console.error('Change avatar failed: Missing required fields');
```

Comment:
I am taking this as an example, but this applies to nearly all of the API endpoint loggings.
If we do it cleanly, I'd prefer to log user/client problems as warnings, not errors. Error means something on the backend went wrong. An unauthorized should be an error. File locked etc should be an error. If the frontend sends invalid data, a field is missing, the name already exists, etc, that's a client issue. It should be logged as an error on the client side, and the backend displays this as a warning. Cause from the backend perspective, there was nothing wrong in handling this.

Reply:
I modified most of the endpoint responses as warnings.